docker-stack.yml

```diff
@@ -1,23 +1,42 @@
 version: '3.4'
 
 services:
-    www:
+    long_lived:
         image: nginx:alpine
         ports:
             - "8000:80"
         deploy:
             replicas: 2
-    success:
+        healthcheck:
+            test: "wget -O - -q http://localhost/"
+            interval: 1s
+            timeout: 2s
+            retries: 10
+    short_lived:
         image: alpine:3.6
-        command: "sh -c 'exit 0'"
+        command: "sh -c 'touch /tmp/healthy && sleep 50 && exit 0'"
         deploy:
-            replicas: 3
+            replicas: 2
+            labels:
+                - 'docker-php.service.lifecycle.shortlived'
             restart_policy:
                 condition: none
-    # failure:
-    #     image: alpine:3.6
-    #     command: "sh -c 'exit 127'"
-    #     deploy:
-    #         replicas: 1
-    #         restart_policy:
-    #             condition: none
+        healthcheck:
+            test: "cat /tmp/healthy"
+            interval: 1s
+            timeout: 2s
+            retries: 10
+
+    long_lived_flapping:
+        image: alpine:3.6
+        environment:
+            - NO_FAILURES_AFTER_TIMESTAMP
+        command: "sh -c 'if [ $$(date +%s) -lt $$NO_FAILURES_AFTER_TIMESTAMP ] ; then sleep 1; echo ho no...; exit 128; else echo no failure mode; touch /tmp/healthy; sleep 86400; echo byebye; exit 0; fi'"
```

Comment:
So many shell 💩 !

Reply:
yup, this is a big shitty one line I know... but still unsure there's a supported way to beautify it.